api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -1067,7 +968,7 @@ private Timestamp expiredBy(Duration duration) {
   }
 
   private void advanceClockDays(long days) {
-    PROVIDED_CLOCK.increment(Duration.ofDays(days).toMillis());
+    PROVIDED_CLOCK.increment(daysPlusSome(days).toMillis());
```

Comment:
the grace period test was actually incorrect because it checked the *exact moment* of expiration rather than slightly after.  This will help future tests like this avoid that issue